recipes/seadex-essentials/all/conanfile.py

```diff
@@ -0,0 +1,97 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.microsoft import check_min_vs, is_msvc
+from conan.tools.files import export_conandata_patches, get, copy
```

Comment:
sneak in some cleanup

```suggestion
from conan.tools.files import get, copy
```